### PR TITLE
NEMS build

### PR DESCRIPTION
**Pull request by zacharyburnettNOAA**
changes made by @pvelissariou1

### CODE REVIEW

**Comment by zacharyburnettNOAA** [comment]
these are the changes made by Takis; I still need to test them to make sure the build compiles.

File: cmake/modules/FindNetCDF.cmake [hunk 1]
```diff
@@ -1,20 +1,29 @@
-#
-# (C) Copyright 2011 ECMWF.
-# (C) Copyright 2020 NOAA
-# (C) Copyright 2021 Helmholtz-Zentrum Hereon
```
Comment (zacharyburnettNOAA) [inline cmake/modules/FindNetCDF.cmake:4]: Not sure if this is necessary, I can revert this specific change if needed

Comment (pvelissariou1) [reply cmake/modules/FindNetCDF.cmake:4]: Zach, there is a more recent FindNetCDF.cmake file, in my repo. please make sure that you are using that one

**Comment by josephzhang8** [comment]
BTW: I've put schism-esmf as a submodule inside schism/thirdparty/

-Joseph

Y. Joseph Zhang
Web: schism.wiki
Office: 804 684 7466

**Comment by zacharyburnettNOAA** [comment]
> 
> BTW: I've put schism-esmf as a submodule inside schism/thirdparty/

Fantastic, thank you! I will work on merging that into my PR, along with the other more recent changes.

File: cmake/modules/FindNetCDF.cmake (hunk 1 above)
Comment (zacharyburnettNOAA) [reply cmake/modules/FindNetCDF.cmake:4]: alright thanks, I will incorporate this

Comment (pvelissariou1) [reply cmake/modules/FindNetCDF.cmake:4]: Zach let's keep the copyright lines on the new one as they are. Originally the file was developed for ECMWF and
now I have put my additions from the NOAA side.

**Comment by zacharyburnettNOAA** [comment]
moving commit head of source branch to local branch `feature/nems` to reduce number of forks - new pull request is #32

**Comment by josephzhang8** [comment]
Hi Zach:

It seems all changes in those 8 files should be rejected b/c they are older than master. I noticed your branch feature/nems has incorporated new changes (good). Let me know if my understanding is correct. Thx.

-Joseph

Y. Joseph Zhang
Web: schism.wiki
Office: 804 684 7466

**Pull request closed without merging**